services/network/BonjourServer.hpp

```diff
@@ -51,7 +51,6 @@ namespace services
 
         private:
             BonjourServer& server;
-            uint16_t queryId;
```

Comment:
if queryId is not used, then it should be removed from the constructor

Reply:
queryId is not used as a member in the class, but it is used in the contstructor to fill in the queryId of DnsRecordHeader, a temporary that is then streamed to stream.